2021-11-27 01:59:56,219 INFO    MainThread:9536 [wandb_setup.py:_flush():71] setting env: {}
2021-11-27 01:59:56,219 INFO    MainThread:9536 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-27 01:59:56,219 INFO    MainThread:9536 [wandb_setup.py:_flush():71] setting login settings: {'api_key': '0f579240b99e6ed291924359c5de323258326760'}
2021-11-27 01:59:56,219 INFO    MainThread:9536 [wandb_init.py:_log_setup():357] Logging user logs to c:\Users\ADMIN\Desktop\Rahav\IIMB\Term5\MLOps\project\MLOps_project\notebooks\wandb\run-20211127_015956-2jn6dsap\logs\debug.log
2021-11-27 01:59:56,219 INFO    MainThread:9536 [wandb_init.py:_log_setup():358] Logging internal logs to c:\Users\ADMIN\Desktop\Rahav\IIMB\Term5\MLOps\project\MLOps_project\notebooks\wandb\run-20211127_015956-2jn6dsap\logs\debug-internal.log
2021-11-27 01:59:56,232 INFO    MainThread:9536 [wandb_init.py:_jupyter_setup():307] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001929FB24670>
2021-11-27 01:59:56,232 INFO    MainThread:9536 [wandb_init.py:init():390] calling init triggers
2021-11-27 01:59:56,232 INFO    MainThread:9536 [wandb_init.py:init():395] wandb.init called with sweep_config: {}
config: {}
2021-11-27 01:59:56,232 INFO    MainThread:9536 [wandb_init.py:init():435] starting backend
2021-11-27 01:59:56,232 INFO    MainThread:9536 [backend.py:_multiprocessing_setup():94] multiprocessing start_methods=spawn, using: spawn
2021-11-27 01:59:56,235 INFO    MainThread:9536 [backend.py:ensure_launched():198] starting backend process...
2021-11-27 01:59:56,347 INFO    MainThread:9536 [backend.py:ensure_launched():203] started backend process with pid: 15488
2021-11-27 01:59:56,355 INFO    MainThread:9536 [wandb_init.py:init():444] backend started and connected
2021-11-27 01:59:56,371 INFO    MainThread:9536 [wandb_run.py:_label_probe_notebook():866] probe notebook
2021-11-27 01:59:56,379 INFO    MainThread:9536 [wandb_run.py:_label_probe_notebook():876] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-11-27 01:59:56,379 INFO    MainThread:9536 [wandb_init.py:init():503] updated telemetry
2021-11-27 01:59:56,467 INFO    MainThread:9536 [wandb_init.py:init():533] communicating current version
2021-11-27 01:59:58,574 INFO    MainThread:9536 [wandb_init.py:init():538] got version response upgrade_message: "wandb version 0.12.7 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-11-27 01:59:58,574 INFO    MainThread:9536 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2021-11-27 01:59:58,942 INFO    MainThread:9536 [wandb_init.py:init():576] starting run threads in backend
2021-11-27 02:00:00,116 INFO    MainThread:9536 [wandb_run.py:_console_start():1693] atexit reg
2021-11-27 02:00:00,124 INFO    MainThread:9536 [wandb_run.py:_redirect():1567] redirect: SettingsConsole.WRAP
2021-11-27 02:00:00,124 INFO    MainThread:9536 [wandb_run.py:_redirect():1604] Wrapping output streams.
2021-11-27 02:00:00,124 INFO    MainThread:9536 [wandb_run.py:_redirect():1628] Redirects installed.
2021-11-27 02:00:00,124 INFO    MainThread:9536 [wandb_init.py:init():603] run started, returning control to user process
2021-11-27 02:00:00,180 INFO    MainThread:9536 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-27 02:00:00,180 INFO    MainThread:9536 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-27 02:00:42,369 INFO    MainThread:9536 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-27 02:00:42,377 INFO    MainThread:9536 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-27 02:00:42,377 INFO    MainThread:9536 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-27 02:00:44,473 INFO    MainThread:9536 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-27 02:00:44,473 INFO    MainThread:9536 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-27 02:00:44,473 INFO    MainThread:9536 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-27 02:00:45,994 INFO    MainThread:9536 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-27 02:00:46,042 INFO    MainThread:9536 [wandb_init.py:_pause_backend():270] pausing backend
2021-11-27 02:00:46,042 INFO    MainThread:9536 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-27 02:00:53,507 INFO    MainThread:9536 [wandb_init.py:_resume_backend():279] resuming backend
2021-11-27 02:01:41,243 INFO    MainThread:9536 [wandb_setup.py:_flush():71] setting env: {}
2021-11-27 02:01:41,243 INFO    MainThread:9536 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-27 02:01:41,243 INFO    MainThread:9536 [wandb_setup.py:_flush():71] setting login settings: {'api_key': '0f579240b99e6ed291924359c5de323258326760'}
2021-11-27 02:01:41,243 INFO    MainThread:9536 [wandb_init.py:_log_setup():357] Logging user logs to c:\Users\ADMIN\Desktop\Rahav\IIMB\Term5\MLOps\project\MLOps_project\notebooks\wandb\run-20211127_020141-fr04xp0z\logs\debug.log
2021-11-27 02:01:41,243 INFO    MainThread:9536 [wandb_init.py:_log_setup():358] Logging internal logs to c:\Users\ADMIN\Desktop\Rahav\IIMB\Term5\MLOps\project\MLOps_project\notebooks\wandb\run-20211127_020141-fr04xp0z\logs\debug-internal.log
2021-11-27 02:01:41,243 INFO    MainThread:9536 [wandb_init.py:init():390] calling init triggers
2021-11-27 02:01:41,243 INFO    MainThread:9536 [wandb_init.py:init():395] wandb.init called with sweep_config: {}
config: {}
2021-11-27 02:01:41,243 INFO    MainThread:9536 [wandb_init.py:init():410] re-initializing run, found existing run on stack: 2jn6dsap
2021-11-27 02:01:41,248 INFO    MainThread:9536 [wandb_run.py:finish():1347] finishing run rahav-manoharan/mlops-project/2jn6dsap
2021-11-27 02:01:41,345 INFO    MainThread:9536 [jupyter.py:save_history():429] not saving jupyter history
2021-11-27 02:01:41,345 INFO    MainThread:9536 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-27 02:01:41,345 INFO    MainThread:9536 [wandb_init.py:_jupyter_teardown():289] cleaning up jupyter logic
2021-11-27 02:01:41,345 INFO    MainThread:9536 [wandb_run.py:_atexit_cleanup():1663] got exitcode: 0
2021-11-27 02:01:41,345 INFO    MainThread:9536 [wandb_run.py:_restore():1635] restore
2021-11-27 02:01:41,871 INFO    MainThread:9536 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1187
  total_bytes: 1187
}

2021-11-27 02:01:42,298 INFO    MainThread:9536 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1187
  total_bytes: 1187
}

2021-11-27 02:01:42,718 INFO    MainThread:9536 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1187
  total_bytes: 5435
}

2021-11-27 02:01:42,825 INFO    MainThread:9536 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1187
  total_bytes: 5435
}

2021-11-27 02:01:42,930 INFO    MainThread:9536 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1187
  total_bytes: 5435
}

2021-11-27 02:01:43,050 INFO    MainThread:9536 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1187
  total_bytes: 5435
}

2021-11-27 02:01:43,158 INFO    MainThread:9536 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1187
  total_bytes: 5435
}

2021-11-27 02:01:43,258 INFO    MainThread:9536 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2363
  total_bytes: 5435
}

2021-11-27 02:01:43,370 INFO    MainThread:9536 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5435
  total_bytes: 5435
}

2021-11-27 02:01:43,481 INFO    MainThread:9536 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5435
  total_bytes: 5435
}

2021-11-27 02:01:43,599 INFO    MainThread:9536 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5435
  total_bytes: 5435
}

2021-11-27 02:01:43,707 INFO    MainThread:9536 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5435
  total_bytes: 5435
}

2021-11-27 02:01:43,807 INFO    MainThread:9536 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5435
  total_bytes: 5435
}

2021-11-27 02:01:43,916 INFO    MainThread:9536 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5435
  total_bytes: 5435
}

2021-11-27 02:01:44,389 INFO    MainThread:9536 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5435
  total_bytes: 5435
}

2021-11-27 02:01:45,271 INFO    MainThread:9536 [wandb_run.py:_wait_for_finish():1793] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5435
  total_bytes: 5435
}
local_info {
}

2021-11-27 02:01:46,380 INFO    MainThread:9536 [wandb_run.py:_append_files():2061] logging synced files
